po/ber.po

```diff
@@ -7,7 +7,7 @@ msgid ""
 msgstr ""
```

Comment:
Just a nit, but normal commit message for this is `i18n: Update translation templates`.